api/v1alpha1/securitypolicy_types.go

```diff
@@ -104,12 +111,45 @@ type AccessDefinition struct {
 	// TODO: APIID should not really be needed, as is auto-set from the APIDefnition Resource
 	APIID    string   `json:"api_id,omitempty"`
 	Versions []string `json:"versions,omitempty"`
-	// RestrictedTypes []graphql.Type `json:"restricted_types"`
+
+	// Field access of GraphQL APIs can be restricted by setting up an allowed types list in a policy
+	// or directly on a key. Please see https://tyk.io/docs/graphql/field-based-permissions/ for more details.
```

Comment:
Do we need to specify URL to docs here?

Reply:
Removed in b2095bfd75716efbff6aefa8d29c898d66ade395 👍